components/badge/badge_input.c

```diff
@@ -76,6 +77,7 @@ badge_input_add_event(uint32_t button_id, bool pressed, bool in_isr)
 				ets_printf("badge_input: input queue full.\n");
 			}
 		}
+		badge_vibrator_buzz(70000);
```

Comment:
I just realised it probably makes more sense to do haptic feedback from the mpr121 and cpt112s handlers in components/badge/badge.c.

It solves two issues: both handlers are never called in interrupt handlers and with the current boards, if there's a mpr121 or cpt112s, there's always a pin reserved for the vibrator motor.

But I would still suggest to make it optional.